airbyte/_executors/docker.py

```
@@ -76,6 +77,11 @@ def map_cli_args(self, args: list[str]) -> list[str]:
                 # This is a file path and we need to map it to the same file within the
                 # relative path of the file within the container's volume.
                 for local_volume, container_path in self.volumes.items():
+                    # If the container path corresponds to the container's temporary directory,
+                    # then the local temporary directory path is used (as the local volume
+                    # path can be overridden).
+                    if container_path == DEFAULT_AIRBYTE_CONTAINER_TEMP_DIR:
+                        local_volume = TEMP_DIR
```

Ruff (0.8.2)

`for` loop variable `local_volume` overwritten by assignment target

(PLW2901)